doc/source/examples/extended_examples/static_rom/static_rom_data_generation.py

```diff
@@ -0,0 +1,302 @@
+    scoped_result = result.on_last_time_freq.on_mesh_scoping(scoping)
+
+    # Result must be sorted by scoping to ensure consistency across outputs.
+    sorted_result = dpf.operators.logic.ascending_sort_fc(
+        scoped_result, sort_by_scoping=True
+    )
+    result_field = sorted_result.outputs.fields_container()[0]
+
+    if new_metadata:
+        write_points(model, scoping, output_folder)
+        write_doe_headers(output_folder, name, parameters)
+        write_settings(output_folder, result_field, name, is_deformation=is_deformation)
+
+    # Write snapshots
+    snapshot_folder = output_folder.joinpath("snapshots")
+    snapshot_folder.mkdir(parents=True, exist_ok=True)
+    snap_name = f"file{snap_idx}.bin"
+    write_doe_entry(output_folder, snap_name, parameters)
+    write_binary(snapshot_folder.joinpath(snap_name), result_field.data)
+
+
+def export_static_ROM_data(
+    mapdl_results: list[tuple[str, dict]], output_folder: str | Path
+):
+    """
+    Export static ROM data to output folder.
+
+    Parameters
+    ----------
+    mapdl_results: list[tuple[str, dict]]
+        list of tuples of MAPDL result file path and the parameter values for each variation solved.
+    output_folder: str|Path
+        location where ROM output data will be stored.
+    """
+    for idx, (rst_path, parameters) in enumerate(mapdl_results):
+        # Load the results to DPF and create scoping.
+        model = dpf.Model(rst_path)
+        scoping = get_scoping(model)
+
+        # Only create points.bin and settings.json on first design point.
+        new_metadata = idx == 0
+
+        # Export displacement and stress data.
+        for name in ["displacement", "stress"]:
+            data_folder = Path(output_folder).joinpath(name)
+            export_static_ROM_variation(
+                model,
+                scoping,
+                name,
+                data_folder,
+                parameters=parameters,
+                snap_idx=idx,
+                new_metadata=new_metadata,
+            )
+
+
+def run_mapdl_variations():
+    """
+    Run the MAPDL model parametric variations.
+
+    Returns
+    -------
+    list[tuple[str, dict]]
+        list of tuples of MAPDL result file path (on the platform where MAPDL was executed) and the
+        parameter values for each variation solved.
+    """
+    # Specify the force load variations
+    forces = [250, 500, 750, 1000]
+
+    # First, start MAPDL and disable all but error messages.
+    mapdl = launch_mapdl(loglevel="ERROR")
+
+    # Download the example database: ``notch_file`` is the path to the downloaded file.
+    notch_file = download_example_data(
+        filename="3d_notch.db", directory="pymapdl/static_ROM_data_generation"
+    )
+
+    mapdl.resume(notch_file, mute=True)
+
+    # Initialise the outputs
+    outputs = []
+
+    # Solve the parameter variations.
+    for idx, force_load in enumerate(forces):
+        # Rename the job and change log, err etc. files.
```

Comment:
```suggestion
        # Rename the job, change log, and error log files
```
Avoid Latin abbreviations.

Reply:
Noted and done.